Dockerfile

```diff
@@ -68,7 +77,7 @@ RUN COMMIT_SHA=f546575109111c455354861a0567c8aa794208a2 && \
     rm -rf ../tests test ../tensorflow_binding
 
 # uninstall stuff from base container
-RUN pip uninstall -y sacrebleu
+RUN pip uninstall -y sacrebleu torchtext
```

Comment:
Wait, torchtext is being deleted here again, this is unecessary now that we built from source.

Reply:
I moved it earlier now.